tests/main.c

```diff
@@ -372,6 +372,22 @@ void test_stringContains() {
     printGreen("test_stringContains\n");
 }
 
+void test_stringToDouble() {
+    TString s1 = stringInitWithCharArr("3.14159");
+    TString s2 = stringInitWithCharArr("123");
+    TString s3 = stringInitWithCharArr("-100");
+
```

Comment:
Add test with empty string or invalid double `-12pas7 8.14`

P.S. you forgot to call this test in `main()`